extra/setup.py

```diff
@@ -46,9 +46,9 @@ def create_version_file():
 
 requirements = [
     'numpy>=1.16.0',
-    'gluoncv==0.9.0',
+    'gluoncv==0.10.0b20201216',
```

Comment:
Why is gluoncv present twice in this requirements list?

Reply:
Also, how necessary is it to be using pre-releases from gluoncv? What if v0.10 isn't released by the time we want to release v0.1 AG?